database/src/rocks.rs

```diff
@@ -25,7 +25,7 @@ impl Database for RocksDB {
     fn get(&self, key: &[u8]) -> Option<Vec<u8>> {
         self.db
             .get(key)
-            .unwrap_or_else(|e| panic!("unrecoverable database error {}", e))
+            .unwrap_or_else(|e| panic!("unrecoverable database error {}", e)) //TODO: this is probably not the right thing to do when handling a abci or REST query. Perhaps we should have a must_get method?
```

Comment:
How it would differ from this `get`?

Reply:
We should discuss as part of a separate issue. I added it here when I cam across it.